packages/apps/tools/src/utils/network.ts

```diff
@@ -22,7 +22,7 @@ export interface INetworkData {
 }
 
 const isNetwork = (x: any): x is Network =>
-  ['mainnet01', 'testnet04', 'testnet05'].includes(x);
+  ['mainnet01', 'testnet04'].includes(x);
 
 export const getConfigNetworkNames = (): DefinedNetwork[] => {
   return Object.keys(kadenaConstants).filter((key) =>
```

Comment:
Get rid of this in favor of using `networkNames` from packages/apps/tools/src/constants/kadena.ts